frontend/packages/process-editor/src/components/ConfigPanel/EditDataTypesToSign/SelectDataTypesToSign/SelectDataTypesToSign.module.css

```diff
@@ -0,0 +1,19 @@
+.buttons {
+  display: flex;
+  gap: var(--fds-spacing-3);
+  margin-top: 30px;
```

Comment:
Can we use a design-token here? 😄

Reply:
This was a hacky solution to align the button with the textarea.

![field-top-aligned (1)](https://github.com/Altinn/altinn-studio/assets/24462611/b1ff6c42-cd87-4e33-8a04-268c2dca6fa6)

I tried using the `fds-spacing` values, but the alignment was not correct. The button was either too high or too low.

The problem here is that the label and the textarea are in the same div generated by the `Combobox` from Designsystem, while the button is outside of the div. Here is how it looks without the `margin-top`:

![field-top (1)](https://github.com/Altinn/altinn-studio/assets/24462611/2ea0636e-bef7-4ee5-8217-d26025b8b3d2)

I just committed a new solution that fixes the alignment by extracting the label from the `Combobox` component. This solution no longer needs a defined `margin-top` and will work regardless of the size of the label.
The main problem with this solution is that we have to define the space between the label and the component, which may differ with the one from design system 🤔

Reply:
I just committed a new solution that fixes the alignment by extracting the label from the `Combobox` component. This solution no longer needs a defined `margin-top` and will work regardless of the size of the label.
The main problem with this solution is that we have to define the space between the label and the component, which may differ with the one from design system 🤔

Reply:
That's great to hear! 😄